### DISCUSSION TITLE
Core Miner keeps restarting

### DISCUSSION THREAD

**Discussion by mpowpow** [catchthatrabbit/coreminer#80] [Miner]
I got everything set up and installed based on the windows guide for Ubuntu LTS and it works but after about 1 minute it just restarts the miner. I am using miner v0.19.81.

**Comment by rastislavcore** [comment]
If you are using the mine.sh script, there is an infinite loop to continue mining in case of some issues. Then I suspect the setup is not done correctly. If you believe that is a bug, please report it as an issue.

**Reply by mpowpow** [reply]
The only warning I get is _"Not able to validate wallet! (Install 'bc' if needed.)"_ because the wallet isn't in the same directory it is on another machine. Could you tell me what bc is, and should I install it?
 
My config is:
```
wallet=cb....
worker=Auri
server[1]=us.catchthatrabbit.com
port[1]=8008
server[2]=us1.catchthatrabbit.com
port[2]=8008
```

**Reply by mpowpow** [reply]
I am using mine.sh

**Reply by rastislavcore** [reply]
The [bc command](https://phoenixnap.com/kb/linux-bc) (short for basic calculator) is a command-line utility that renders the bc language. The program runs as an interactive program or takes standard input to perform arbitrary precision arithmetic.

Please run your mine.sh in debug mode. In short `bash -x script [arg1 ...]`

In case you are still facing difficulties, please open an [Issue](https://github.com/catchthatrabbit/coreminer/issues/new?assignees=raisty&labels=bug&projects=&template=bug_report.yml&title=%5BBUG%5D+).
